src/main/resources/assets/js/application.js

```diff
@@ -0,0 +1,2 @@
+// Check if domain is gov.uk so that we can display the gov.uk branding i.e. font and crest
+document.body.className = (window.location.href.indexOf("openregister-org") >= 0) ? "openregister-org" : "gov-uk";
```

Comment:
I'm not sure this will do what you want?

```
> window.location.href
"https://country.test.openregister.org/"
> window.location.href.indexOf("openregister-org") >= 0
false
```